src/components/Banner/Banner.stories.tsx

```diff
@@ -137,55 +141,59 @@ export const ErrorWithAction: StoryObj<Args> = {
 export const BrandDismissable: StoryObj<Args> = {
   ...Brand,
   args: {
-    dismissable: true,
+    onDismiss: dismissMethod,
```

Comment:
nit: would [Storybook's actions addon](https://storybook.js.org/docs/react/essentials/actions) work?

```
{
  ...Brand,
  argTypes: { onDismiss: { action: 'dismissed' } },
};
```

if so, it's a little nicer since it surfaces messages in a storybook panel

Reply:
It works in storybook but the close button is missing from the snapshots.. 🤔

Reply:
https://github.com/chanzuckerberg/edu-design-system/compare/diedra/banner/dismiss-logic...diedra/banner/dismiss-logic-with-dismiss-actions

Reply:
ahh shoot, I think that is a limitation of the addon 😞  oh well, thanks for trying it out